app/components/screens/products/VM.ts

```diff
@@ -0,0 +1,12 @@
+import Product from "../../../data/product/Product";
+import {NavigationState} from "./State";
+
+export interface VM {
```

Comment:
What is VM and VMMapper?

Reply:
VM stands for ViewModel and VMMapper is a function that maps the screen state+props to the VM. 

The ViewModel is meant to be a representation of the screen UI at any given point of time. 

This pattern is kinda based on the MVVM and MVP architecture and is used heavily in the Android world.

Reply:
Got it. Thanks.